scripts/schema-publish.sh

```diff
@@ -6,15 +6,52 @@
 
 schemaDir="src/schemas/validation"
 branch=$(git branch --show-current)
-version=${branch:1:3}
-echo === Building schemas into ./deploy/oas/$version
+
+if [[ $branch =~ ^v([0-9]+\.[0-9]+)\.[0-9]+ ]]; then
```

Comment:
The branches are named
* v3.1-dev
* v3.2-dev

so the pattern really needs to be
```suggestion
if [[ $branch =~ ^v([0-9]+\.[0-9]+)-dev$ ]]; then
```